tendrl-notifier.spec

```diff
@@ -31,7 +31,7 @@ rm -rf html/.{doctrees,buildinfo}
 
 %install
 %{__python} setup.py install --single-version-externally-managed -O1 --root=$RPM_BUILD_ROOT --record=INSTALLED_FILES
-install -m  0755  --directory $RPM_BUILD_ROOT%{_var}/log/tendrl/notifier
+# install -m  0755  --directory $RPM_BUILD_ROOT%{_var}/log/tendrl/notifier
```

Comment:
this is not required, we are not storing any log in /var/log/tendrl/notifier, so it is not required i feel